site-cookbooks/es_snap/recipes/client.rb

```diff
@@ -0,0 +1,24 @@
+node[:nfs_client][:packages].each do |p|
+    package "#{p}" do
+        action :install
+    end
+end
+
+directory  "/opt/nfs" do
+    owner  'root'
+    mode   '0755'
+    action :create
+end
+
+bash "Mount nfs" do
+    user 'root'
+    code <<-EOS
+    nfs_server_ip="#{node[:nfs_server][:address]}"
+    df -h | grep nfs
```

Comment:
bash nitpick: `if df -h | grep -q nfs`. Using -q for a boolean grep is good because it exits after the first match. 

I see that chef has a mount resource type. Will that work here?